pkg/manager/cluster.go

```diff
@@ -81,20 +81,11 @@ func (c *Cluster) GenerateClusterBundle(bundleDir string) (string, error) {
 func (c *Cluster) generateSupportBundleYAMLs(yamlsDir string, errLog io.Writer) {
 	// Cluster scope
 	globalDir := filepath.Join(yamlsDir, "cluster")
-	c.generateKubernetesClusterYAMLs(globalDir, errLog)
 	c.generateDiscoveredClusterYAMLs(globalDir, errLog)
 
-	// Namespaced scope: k8s resources
+	// Namespaced scope: all resources
```

Comment:
Separate this part into another commit, since it's not related to group version etc change.